File: NumberRamdom/app/src/main/java/com/example/numberramdom/MainActivity.java
package com.example.numberramdom;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

import java.util.Random;

public class MainActivity extends AppCompatActivity {
    TextView txt;
    Button btn ;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        anhXa();
        getNumberRandom();
    }
    private void getNumberRandom(){
        btn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Random ram = new Random();
                int number = ram.nextInt(10);
                txt.setText("Your Number :" + number);
            }
        });
    }
    private void anhXa() {
        txt = findViewById(R.id.textNumber);
        btn = findViewById(R.id.btnNumber);
    }

}

File: Caculator/app/src/main/java/com/example/caculator/MainActivity.kt
package com.example.caculator

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        btnCong.setOnClickListener(){
            var so1:Double = (etNumber1.text.toString()).toDouble()
            var so2:Double = (etNumber2.text.toString()).toDouble()
            var rs:Double = so1 + so2
            txtKetqua.text = rs.toString()
        }

        btnTru.setOnClickListener(){
            var so1:Double = (etNumber1.text.toString()).toDouble()
            var so2:Double = (etNumber2.text.toString()).toDouble()
            var rs:Double = so1 - so2
            txtKetqua.text = rs.toString()
        }

        btnNhan.setOnClickListener(){
            var so1:Double = (etNumber1.text.toString()).toDouble()
            var so2:Double = (etNumber2.text.toString()).toDouble()
            var rs:Double = so1 * so2
            txtKetqua.text = rs.toString()
        }

        btnChia.setOnClickListener(){
            var so1:Double = (etNumber1.text.toString()).toDouble()
            var so2:Double = (etNumber2.text.toString()).toDouble()
            var rs:Double = so1 / so2
            txtKetqua.text = rs.toString()
        }


    }

}